pages/onboarding/index.tsx

```diff
@@ -15,8 +67,77 @@ const Onboarding: NextPage = () => {
           </Button>
         </Link>
       </Box>
+
+      <form onSubmit={handleSubmit}>
+        <TextField
+          label="Bio"
+          name="bio"
+          value={formData.bio}
+          onChange={handleChange}
+          multiline
+          rows={4}
+          fullWidth
+          inputProps={{ maxLength: 1000 }}
+          required
+        />
+        <TextField
```

Comment:
placeholders i think would be helpful here, e.g. so users don't also include the `@`

Reply:
ok, lmk what you think of the placeholders... didn't know what to put

Reply:
placeholders should guide what the users should type in. i would make some (obviously) fake examples.  e.g. for linkedin we want the full URL so maybe something like `https://www.linkedin.com/in/your-profile`